csm/cli/support_bundle/csm_bundle_generate.py

```diff
@@ -111,16 +113,58 @@ class GenerateCsmBundle:
     '''
     @staticmethod
     def generate_bundle(args):
+
         Conf.load(const.CONSUMER_INDEX, args['config_url'])
         log_path = Conf.get(const.CONSUMER_INDEX, const.CSM_LOG_PATH_KEY)
-        csm_log_path = os.path.join(log_path,'csm')
+        csm_log_path = os.path.join(log_path, const.CSM_COMPONENT_NAME)
         bundle_id = args['bundle_id']
         target_path = args ['target']
-        services = args['services'] #Not making any use of service option for now
-        target_path = os.path.join(target_path, bundle_id)
+        duration = args['duration']
+        size_limit = args['size_limit']
+        # services = args['services'] #Not making any use of service option for now
+        # binlogs = args['binlogs'] #Not making any use of binlogs option for now
+        # coredumps = args['coredumps'] #Not making any use of coredumps option for now
+        # stacktrace = args['stacktrace'] #Not making any use of stacktrace option for now
+        target_path = os.path.join(target_path, const.CSM_COMPONENT_NAME)
         os.makedirs(target_path,exist_ok=True)
+
+        csm_size_filtered_logs_dir = os.path.join(const.CSM_SETUP_LOG_DIR,
+            f"{const.CSM_COMPONENT_NAME}_logs_size")
+        GenerateCsmBundle.__clear_tmp_files(csm_size_filtered_logs_dir)
+        os.makedirs(csm_size_filtered_logs_dir,exist_ok=True)
+        csm_time_filtered_logs_dir = os.path.join(const.CSM_SETUP_LOG_DIR,
+            f"{const.CSM_COMPONENT_NAME}_logs_size_time")
+        GenerateCsmBundle.__clear_tmp_files(csm_time_filtered_logs_dir)
+        os.makedirs(csm_time_filtered_logs_dir,exist_ok=True)
+        FilterLog.limit_size(csm_log_path, csm_size_filtered_logs_dir,
```

Comment:
Comment this and add TODO.

Reply:
Done.
Commented out duration filter as we have a open bug - 
[Support Bundle - Filter logs based on duration not working properly](https://jts.seagate.com/browse/EOS-27275)